src/components/ebay-wizard-stepper/examples/04-interactive/template.marko

```diff
@@ -0,0 +1,29 @@
+class {
+    onCreate(input) {
+        this.state = {
+            current: 0
+        };
+        this.items = ['Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5']
+    }
+    prev() {
+        if (this.state.current >= 0) {
+            this.state.current--;
+        }
+    }
+    next() {
+        if (this.state.current < this.items.length) {
+            this.state.current++;
+        }
+    }
+}
+
+<div>
+    <ebay-button on-button-click('prev')>Previous</ebay-button>
```

Comment:
This example is nice. Storybook can also help us achieve something similar with "knobs", right? Mental note for us to look into that again at some point.